### PR TITLE
Notebook documentation

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: .github/workflows/test-notebooks.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+jobs:
+  nbmake:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 3
+      matrix:
+        os: [ubuntu-18.04, macos-10.15]
+
+    steps:
+    - uses: actions/checkout@v1
```
Comment (dzenanz) [inline .github/workflows/test-notebooks.yml:14]: both v1 and v2?

File: examples/ConvertScancoVolumesToOpenStandardFormats.ipynb [hunk 2]
```diff
@@ -0,0 +1,1170 @@
...
+       "        [0., 0., 1.]])}"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "image_nifti = itk.imread('image.mha')\n",
```
Comment (dzenanz) [inline examples/ConvertScancoVolumesToOpenStandardFormats.ipynb:438]: Should this be `.nii.gz`?

**Review by dzenanz — COMMENTED** [review]
mostly looks good

File: .github/workflows/test-notebooks.yml (hunk 1 above)
Comment (thewtex) [reply .github/workflows/test-notebooks.yml:14]: Good catch-- we will bump to v2.

File: examples/ConvertScancoVolumesToOpenStandardFormats.ipynb (hunk 2 above)
Comment (thewtex) [reply examples/ConvertScancoVolumesToOpenStandardFormats.ipynb:438]: Yes, good catch.

Comment (thewtex) [reply examples/ConvertScancoVolumesToOpenStandardFormats.ipynb:438]: This revealed that Nifti is not preserving the metadata -- I will remove it from the notebook.

**Pull request merged**